README.md

```diff
@@ -212,6 +212,27 @@ model = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True)
 outputs = model.generate(inputs, streamer=streamer, max_new_tokens=300)
 ```
 
+You can also load your quantized low bit model(GPTQ/AWQ/RTN/AutoRound) by the below code.
+```python
+from transformers import AutoTokenizer, TextStreamer
+from intel_extension_for_transformers.transformers import AutoModelForCausalLM, WeightOnlyQuantConfig
+
+# Download Hugging Face GPTQ/AWQ/AutoRound model to local path
+model_name = "PATH_TO_MODEL"  # local path to model
+woq_config = WeightOnlyQuantConfig(use_gptq=True)   # use_awq=True for awq model
```

Comment:
autoround?

Reply:
Ok, I will add auto round args

Reply:
Download Hugging Face GPTQ/AWQ model or generate model to local

Reply:
fixed on https://github.com/intel/intel-extension-for-transformers/pull/1196/commits/b12655d8247d93b81cd5297655badedf687ccbfd